spec/support/api/schemas/notification.json

```diff
@@ -0,0 +1,19 @@
+{
+	"properties" : {
```

Comment:
The spacing on this JSON is very large, can you make sure you're only using 2 spaces instead of tabs/4+ spaces?